packages/epinio/epinio/generated-changes/patch/Chart.yaml.patch

```diff
@@ -0,0 +1,28 @@
+--- charts-original/Chart.yaml
++++ charts/Chart.yaml
+@@ -1,5 +1,16 @@
+ annotations:
+   artifacthub.io/license: Apache-2.0
++  catalog.cattle.io/auto-install: epinio-crd=match
++  catalog.cattle.io/certified: rancher
++  catalog.cattle.io/display-name: Epinio
++  catalog.cattle.io/experimental: "true"
++  catalog.cattle.io/kube-version: '>= 1.20.0-0 < 1.24.0-0'
++  catalog.cattle.io/namespace: cattle-epinio-system
++  catalog.cattle.io/permits-os: linux,windows
++  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <=2.6.99-0'
```

Comment:
Update this to `'>= 2.6.0-0 < 2.7.0-0'`

Reply:
Done. Commits 48ff98d3c8a1d2ada31aed460aeec575db054b2d, 891495b445cbaf4ff1395c87d64c4ae230a672e7